### ISSUE TITLE
Change translation of "diary" to "entry"

### ISSUE THREAD

**Issue by ghost** [krrr/Hazama#4]
When I create a new entry in the diary, then the title of the dialog is "New diary". In my opinion it should be "New entry" (because diary is the whole book).

Also on the command line: 
"INFO: Diary Book (nikkichou.db) with 4 diaries" 
But should be: "...with 4 entries".

**Comment by krrr** [comment]
You are right. I will change it in the C++ version (I'm rewriting this app in C++; it will be ready a few months later)
